### DISCUSSION TITLE
'K_OBJ_DRIVER_GP' undeclared

### DISCUSSION THREAD

**Discussion by sambazley** [zephyrproject-rtos/zephyr#57505] [Q&A]
After setting CONFIG_ARM_MPU=y, I get `kobject_prebuilt_hash.gperf:25:39: error: 'K_OBJ_DRIVER_GP' undeclared here (not in a function); did you mean 'K_OBJ_DRIVER_W1'?`

Presumably K_OBJ_DRIVER_GP should be K_OBJ_DRIVER_GPIO? Could this be caused by something I've done, or is this a bug? Please let me know if there is any more information I need to provide.

I'm using v3.3.0, but can't reproduce with main because 2815f964404a32200e5dbc517e3a1c5822b7b9ae removes gpio_keys.

**Comment by yashi** [comment]
Would you please give us all the steps to reproduce, including all commands you typed and all outputs?

**Reply by sambazley** [reply]
Unfortunatly I'm not sure what exactly I've done to cause the problem. `CONFIG_ARM_MPU=y` was the only thing I changed to cause the build to fail. I appreciate there's not a lot of information to go off, but I was hoping someone else would have some ideas of what may have gone wrong.

**Reply by yashi** [reply]
If you can't reproduce it, we can't.  Without it it's hard to help you.  Please help us to help you with a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example).
